docs/howto/assets/eks/eks-worker-al2.json

```diff
@@ -10,7 +10,8 @@
     "aws_secret_access_key": "{{env `AWS_SECRET_ACCESS_KEY`}}",
     "aws_session_token": "{{env `AWS_SESSION_TOKEN`}}",
 
-    "krustlet_version": "0.1.0",
+    "krustlet_version": "v0.1.0",
+    "krustlet_src": "https://github.com/deislabs/krustlet/archive/{{ user `krustlet_version`}}.tar.gz",
```

Comment:
Could we perhaps leave `krustlet_version` defaulting to `0.1.0` but put the `v` in `krustlet_src`? Currently all the version number does is end up in the AMI tag, but it would be more consistent with how it works for the official EKS AMI.

Reply:
+1 on @peterhuene's suggestion. The `v` is just a vanity thing for human-readability. `0.1.0` is technically the correct version number, so the `v` should be part of the source URL and the version should be `0.1.0`. I don't imagine anyone would tag something like `mytag` in their fork (and if so they can modify the JSON in their fork).